test/native/MODCLUSTER-785/testit.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+source ../includes/script.bash
+
+# check IMG...
+if [ -z ${IMG} ]; then
+  echo "IMG needs to defined, please try"
+  echo "export IMG=quay.io/${USER}/tomcat_mod_cluster"
+  exit 1
+fi
```

Comment:
Don't we want to include these tests in the global testsuite too?

Then with the [latest](https://github.com/modcluster/mod_proxy_cluster/commit/1625f7846fa42bc520bd73738317e47fa1fad1db) changes this would not be needed. I not I would suggest to use the same approach (i.e., if not defined, we should go with the default and inform the user).